src/main/kotlin/oop/Adapter/EURobotAdapter.kt

```diff
@@ -0,0 +1,20 @@
+package oop.Adapter
+
+interface EuropeanRobotTarget
+{
+  var speedInKilometersPerHour: Double
+  fun jump(meters: Double)
+}
+
+data class EURobotAdapter(var usRobot: USRobot) : EuropeanRobotTarget{
+  override var speedInKilometersPerHour: Double
+    get() = usRobot.speedInMilesPerHour * 1.6093
+    set(value) {usRobot.speedInMilesPerHour = value * 0.62137}
```

Comment:
I really like this solution 👍 You are giving behaviour to the getter/setter in a concise way. This solution wouldn't be compatible with immutability, as I suggest in a forward comment, but it's really good.

The only thing I see it could be improved is to store the numbers into a variable with name to improve readability.